### DISCUSSION TITLE
Elasticsearch: Grafana should help when adding lucene query in logs query field in Explore

### DISCUSSION THREAD

**Discussion by marefr** [grafana/grafana#58403] [Data Sources]
**What would you like to be added**:
Would be nice to get some help when adding lucene query in the Elasticsearch logs query field in Explore. For example, being able to select available fields as you type or in a select component and/or suggesting lucene operators for equals, not equals, exists etc.

**Why is this needed**:
Should make it easier to explore what fields there are available for filtering and to write proper formatted lucene queries.

It was discussed/thought of a bit initially when implementing #17605 so adding this so that we can see if others are interested in this feature.

**Comment by torkelo** [comment]
Important to do slate upgrade before this.

**Comment by aocenas** [comment]
This should be probably done together with https://github.com/grafana/grafana/issues/17907

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
